server/controller/src/main/java/ai/starwhale/mlops/domain/job/split/JobSpliteratorImpl.java

```diff
@@ -16,20 +16,19 @@
 
 package ai.starwhale.mlops.domain.job.split;
 
-import ai.starwhale.mlops.common.util.BatchOperateHelper;
 import ai.starwhale.mlops.domain.job.JobDao;
 import ai.starwhale.mlops.domain.job.bo.Job;
 import ai.starwhale.mlops.domain.job.spec.JobSpecParser;
 import ai.starwhale.mlops.domain.job.spec.StepSpec;
 import ai.starwhale.mlops.domain.job.status.JobStatus;
-import ai.starwhale.mlops.domain.job.step.mapper.StepMapper;
+import ai.starwhale.mlops.domain.job.step.StepService;
```

Comment:
what is the benefit of forcing JobSpliterator to depend on service level, do we have a proper abstraction of service layer. 
What is the benefit of increasing hierarchy in our code withou proper abstractions.

Reply:
other domain's mapper should not directly used by other domain